packages/components/src/components/radio-button/_radio-button.scss

```diff
@@ -136,6 +148,7 @@
     .#{$prefix}--radio-button__appearance {
     outline: 2px solid $focus;
     outline-offset: 1.5px;
```

Comment:
Not included in your changes, but while reviewing I noticed that Firefox is calculating this incorrectly and causing a weird focus. 

![image](https://user-images.githubusercontent.com/11928039/157701499-db053538-da55-4846-b7d4-e842d8d9d3e9.png)

Can we just change this to 1px?